profilehooks.py

```diff
@@ -365,29 +375,36 @@ def print_stats(self):
             stats.dump_stats(self.filename)
         if self.stdout:
             funcname, filename, lineno = _identify(self.fn)
-            print("")
-            print("*** PROFILER RESULTS ***")
-            print("%s (%s:%s)" % (funcname, filename, lineno))
+            print_f = print
+            if self._stdout_is_fp:
+                print_f = functools.partial(print, file=self.stdout)
+
+            print_f("")
```

Comment:
FWIW if you're curious why I had `print("")` here instead of a simple `print()`, I had this silly habit of writing code compatible with Python 2 and 3 at the same time without relying on `from __future__ import print_function`.  `print("")` works fine on both Pythons, `print()` without the future import prints the representation of an empty tuple on Python 2.

Anyway, the future import was added a long time ago and I forgot to remove the now-unnecessary `""`s.  They can stay, they don't bother me too much.